cmd/create.go

```diff
@@ -16,10 +16,11 @@ import (
 )
 
 var (
-	componentBinary string
-	componentGit    string
-	componentLocal  string
-	componentPorts  []string
+	componentBinary  string
+	componentGit     string
+	componentLocal   string
+	componentPorts   []string
+	componentEnvVars []string
 )
 
 var componentCreateCmd = &cobra.Command{
```

Comment:
it will be nice to add env example

Reply:
added here https://github.com/redhat-developer/odo/pull/626/files#diff-452662b1e58d63509f34fc4bd164905bR52